src/postgres/inventory/conditionToSql.ts

```diff
@@ -25,7 +25,12 @@ export default function conditionToSql (condition: Condition, path: Array<string
     case 'GREATER_THAN':
       return sql.query`(${sql.identifier(...path)} > ${sql.value(condition.value)})`
     case 'REGEXP':
-      return sql.query`regexp_matches(${sql.identifier(...path)}, ${sql.value(condition.regexp.source)}, ${sql.value(condition.regexp.flags)})`
+      // Parse out the regular expression. In Node.js v4 we can’t get the
+      // `flags` property so we need to do it this way.
+      const match = condition.regexp.toString().match(/^\/(.*?)\/(g?i?m?u?y?)$/i)
```

Comment:
True. This was a really unfortunate incompatibility to discover. What I’d really like to see is an ES5 polyfill that implements `flags` that we can check our implementation against.

Ironically though, I don’t think this even matters because this part of the code never gets run :blush:. It’s for the advanced conditions feature that we don’t yet have.